service.toml

```diff
@@ -32,4 +36,7 @@ type = "string"
 type = "string"
 
 [infos.object.meta.storage-class]
-type = "string"
\ No newline at end of file
+type = "string"
+
+[infos.object.meta.sse_customer_key_sha256]
+type = "string" 
```

Comment:
What's the content of `sse_customer_key_sha256`, a base64 string?

Reply:
base64-encoded SHA-256 hash of the customer-supplied encryption key for the object